Makefile.work

```diff
@@ -76,11 +76,16 @@ ifeq ($(PLATFORM_ARCH),)
 	override PLATFORM_ARCH = $(CONFIGURED_ARCH)
 endif
 
+ifeq ($(BLDENV), buster)
+SLAVE_DIR = sonic-slave-buster
+else
 ifeq ($(BLDENV), stretch)
```

Comment:
Suggest to combine these into one line, e.g, `else ifeq ($(BLDENV), stretch)` and remove the extra `endif` on line 87. I believe this syntax should be supported by our version of make.

Reply:
thansk. resolved. will push.